python/xorbits/_mars/utils.py

```diff
@@ -356,44 +356,53 @@ def __get__(self, obj, owner):
         return self.f(owner)
 
 
+class LazyModule:
+    def __init__(self, name, rename, globals_, locals_):
+        self.name = name
+        self._on_loads = []
+        self._rename = rename
+        self._globals = globals_
+        self._locals = locals_
+
+    def __getattr__(self, item):
+        if item.startswith("_pytest") or item in ("__bases__", "__test__"):
+            raise AttributeError(item)
+
+        real_mod = importlib.import_module(self.name)
+        if self._rename in self._globals:
+            self._globals[self._rename] = real_mod
+        elif self._locals is not None:
+            self._locals[self._rename] = real_mod
+        ret = getattr(real_mod, item)
+        for on_load_func in self._on_loads:
+            on_load_func()
+        # make sure on_load hooks only executed once
+        self._on_loads = []
+        return ret
+
+    def __eq__(self, other: "LazyModule"):
```

Comment:
Looks like finally we did not use this method to do equal assertion?

Reply:
Seems this method is never used for equality assertion since we never use '==' to check if two modules are equivalent.

Previously we use 'is' to check if two names point to the same object.

Reply:
Yeah, so maybe removing it? It's already in there before this PR?

Reply:
Sure, we can remove it.